disco_aws_automation/disco_rds.py

```diff
@@ -39,93 +40,239 @@
 }
 
 
-class DiscoRDS(object):
-    """Class for doing RDS operations on a given environment"""
+class RDS(threading.Thread):
+    """Class for spinning up a single rds instance"""
 
-    def __init__(self, vpc):
+    def __init__(self, env_name, database_identifier, rds_security_group_id, subnet_ids, domain_name):
         """Initialize class"""
+        threading.Thread.__init__(self, name=database_identifier)
+        self.client = boto3.client('rds')
+        self.vpc_name = env_name
```

Comment:
nitpick: looks like we can just derive the vpc_name from the database_identifier without having an extra argument to the method.